src/main/java/com/puppycrawl/tools/checkstyle/DefaultConfiguration.java

```diff
@@ -103,6 +103,11 @@ public String getProperty(String propertyName) throws CheckstyleException {
         return propertyMap.get(propertyName);
     }
 
+    @Override
+    public String getPropertyNoError(String propertyName) {
```

Comment:
This is needed because `getProperty` throws an error if the property doesn't exist, but we need some way to check/retrieve it.